### PR TITLE
Code clean up

### PR DESCRIPTION
**Pull request by tysongach**
- Update SCSS-Lint config to 0.47
- Remove usage of transition and transform Bourbon mixins
- Clean up usage of Bourbon's position mixin
- Remove unit on zero values
- Use zero convention instead of none for borders
- Add leading zeros
- Use double colons for pseudo elements
- ABC properties

### CODE REVIEW

File: source/stylesheets/refills/_pagination.scss [hunk 1]
```diff
@@ -48,6 +47,7 @@
       outline: none;
       padding: ($base-spacing / 4) ($gutter / 2);
       text-decoration: none;
+      transition: all 0.2s ease-in-out;
```
Comment (tysongach) [inline source/stylesheets/refills/_pagination.scss]: +1, but I didn’t want to change the value in this PR; just :abc:’ing.

File: source/stylesheets/refills/_maps.scss [hunk 2]
```diff
@@ -13,8 +13,8 @@
 
 #map_canvas .firstHeading {
   font-size: 1em;
-  line-height: 1em;
   font-weight: bold;
+  line-height: 1em;
```
Comment (tysongach) [inline source/stylesheets/refills/_maps.scss]: +1, but I didn’t want to change the value in this PR; just :abc:’ing.

File: source/stylesheets/refills/_maps.scss [hunk 3]
```diff
@@ -3,7 +3,7 @@
   width: 100%;
 }
 
-#map_canvas img { 
+#map_canvas img {
```
Comment (tysongach) [inline source/stylesheets/refills/_maps.scss]: Out of scope for this PR…

File: source/stylesheets/refills/_grid-items.scss [hunk 4]
```diff
@@ -35,11 +34,12 @@
     cursor: pointer;
     float: left;
     height: $grid-item-height;
-    overflow: hidden;
     outline: none;
+    overflow: hidden;
     padding: 2em;
     text-align: center;
     text-decoration: none;
+    transition: all 0.2s ease-in-out;
```
Comment (tysongach) [inline source/stylesheets/refills/_grid-items.scss]: Out of scope for this PR…

File: source/stylesheets/refills/_hover-tile-animation.scss [hunk 5]
```diff
@@ -24,16 +24,16 @@
...
   }
 
   .hover-tile {
-    @include transition(all, 0.2s ease-in-out);
     background: inherit;
     color: white;
     height: inherit;
     overflow: hidden;
     padding: $base-spacing;
+    transition: all 0.2s ease-in-out;
```
Comment (tysongach) [inline source/stylesheets/refills/_hover-tile-animation.scss]: Out of scope for this PR…

File: source/stylesheets/refills/_grid-items-lines.scss [hunk 6]
```diff
@@ -19,19 +19,19 @@
...
     border-right: $grid-item-border;
     cursor: pointer;
     float: left;
     height: $grid-item-height;
-    overflow: hidden;
     outline: none;
+    overflow: hidden;
     padding: 2em;
     text-decoration: none;
+    transition: all 0.2s ease-in-out;
```
Comment (tysongach) [inline source/stylesheets/refills/_grid-items-lines.scss]: Out of scope for this PR…

File: source/stylesheets/refills/_flex-boxes.scss [hunk 7]
```diff
@@ -37,12 +36,13 @@
     margin: $flex-box-gutter;
     padding: 2em 2em 3em 2em;
     text-decoration: none;
+    transition: all 0.2s ease-in-out;
```
Comment (tysongach) [inline source/stylesheets/refills/_flex-boxes.scss]: Out of scope for this PR…

File: source/stylesheets/refills/_centered-navigation.scss [hunk 8]
```diff
@@ -70,12 +70,12 @@
   }
 
   ul.centered-navigation-menu {
-    -webkit-transform-style: preserve-3d; // stop webkit flicker
     clear: both;
     display: none;
     margin: 0 auto;
     overflow: visible;
     padding: 0;
+    -webkit-transform-style: preserve-3d; // stop webkit flicker
```
Comment (tysongach) [inline source/stylesheets/refills/_centered-navigation.scss]: Out of scope for this PR…

File: source/stylesheets/refills/_fade-in.scss [hunk 9]
```diff
@@ -29,6 +29,6 @@
 }
 
 .js-fade-element-show {
-  @include transition (all 0.4s ease-in-out);
   opacity: 1;
+  transition: all 0.4s ease-in-out;
```
Comment (tysongach) [inline source/stylesheets/refills/_fade-in.scss]: Out of scope for this PR…

File: source/stylesheets/refills/_centered-navigation.scss [hunk 10]
```diff
@@ -189,7 +189,7 @@
         margin-right: $centered-navigation-submenu-padding;
       }
 
-      &:after {
+      &::after {
```
Comment (tysongach) [inline source/stylesheets/refills/_centered-navigation.scss]: Out of scope for this PR…

File: source/stylesheets/refills/_search-bar.scss [hunk 11]
```diff
@@ -1,4 +1,4 @@
-form.search-bar {  
+form.search-bar {
```
Comment (tysongach) [inline source/stylesheets/refills/_search-bar.scss]: Out of scope for this PR…

File: source/stylesheets/refills/_navigation.scss [hunk 12]
```diff
@@ -183,8 +183,8 @@ header.navigation {
         margin-right: $navigation-submenu-padding;
       }
 
-      &:after {
-        content: '›';
+      &::after {
```
Comment (tysongach) [inline source/stylesheets/refills/_navigation.scss]: Out of scope for this PR…

File: source/stylesheets/refills/_dropdown.scss [hunk 13]
```diff
@@ -93,10 +92,11 @@
     position: absolute;
     right: 0;
     top: $dropdown-distance-from-menu;
+    transition: all 0.2s ease-in-out;
```
Comment (tysongach) [inline source/stylesheets/refills/_dropdown.scss]: Out of scope for this PR…

**Comment by joshuaogle** [comment]
I'm sure that took forever, thanks for the hard work. I don't spot any issues or anything that you missed. :+1: 

Some quick thoughts after going through this:
- There are lots of places `px` are used that could be `em` for more consistency
- I'd love to have the variables inside components be namespaced for easier plucking to a global variables stylesheet
- There's so little Bourbon code, it's tempting to make these standalone
- I'm sure there's more. Thanks for the good start!

**Comment by tysongach** [comment]
@joshuaogle Thanks! This came in very handy: https://github.com/HugoGiraudel/scss-lint-regex

I agree with your thoughts, and should be part of a larger conversation.

**Pull request merged**